airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2387,6 +2387,10 @@ components:
             $ref: "#/components/schemas/JobConfigType"
         configId:
           type: string
+        pagesize:
```

Comment:
should we set a convention here that can be easily reused anywhere else that we want to do pagination? e.g. create a separate object that gets inherited or composed?

Reply:
Created a Pagination object and integrated it with the openapi spec and the code.